tools/pika_exporter/exporter/metrics/rocksdb.go

```diff
@@ -382,4 +382,174 @@ var collectRocksDBMetrics = map[string]MetricConfig{
 			ValueName: "live_blob_file_size",
 		},
 	},
+	// column family stats
+	// 以下所有指标的Help信息都没有修改，需要统一修改
```

Comment:
I think you can refer to [the description of the Column Family Stats in ToplingDB](https://github.com/topling/toplingdb/wiki/Grafana-%E7%9B%91%E6%8E%A7%E6%8C%87%E6%A0%87%E8%AF%B4%E6%98%8E-%E2%80%94%E2%80%94-rocksdb-&-toplingdb-%E9%83%A8%E5%88%86) to modify Help information.

Reply:
![Uploading image.png…]()